src/cosmoscout/Application.cpp

```diff
@@ -687,33 +629,120 @@ void Application::testLoadAllPlugins() {
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
+void Application::openPlugin(std::string const& name) {
+  try {
+
+#ifdef __linux__
+    std::string path = PLUGIN_PATH "lib" + name + ".so";
```

Comment:
```suggestion
    std::string path = PLUGIN_PATH + "lib" + name + ".so";
```

Reply:
This will not compile this way. Both `PLUGIN_PATH` and `"lib"` are string literals which are concatenated automatically (and not with an operator). ISO C:

> In translation phase 6, the multibyte character sequences specified by any sequence of adjacent character and identically-prefixed wide string literal tokens are concatenated into a single multibyte character sequence.

For example, have a look at https://www.techiedelight.com/concatenate-string-literals-c-cpp/ or https://stackoverflow.com/questions/12120944/how-does-concatenation-of-two-string-literals-work.